moto/ec2/responses/vpn_connections.py

```diff
@@ -191,128 +211,20 @@ def describe_vpn_connections(self):
-            <fragmentation_before_encryption>true</fragmentation_before_encryption>
-            <tcp_mss_adjustment>1387</tcp_mss_adjustment>
-            <dead_peer_detection>
-              <interval>10</interval>
-              <retries>3</retries>
-            </dead_peer_detection>
-            </ipsec>
-          </ipsec_tunnel>
-          <ipsec_tunnel>
-            <customer_gateway>
-            <tunnel_outside_address>
-              <ip_address>12.1.2.3</ip_address>
-            </tunnel_outside_address>
-            <tunnel_inside_address>
-              <ip_address>169.254.44.42</ip_address>
-              <network_mask>255.255.255.252</network_mask>
-              <network_cidr>30</network_cidr>
-            </tunnel_inside_address>
-            <bgp>
-              <asn>65000</asn>
-              <hold_time>30</hold_time>
-            </bgp>
-            </customer_gateway>
-            <vpn_gateway>
-            <tunnel_outside_address>
-              <ip_address>52.2.144.13</ip_address>
-            </tunnel_outside_address>
-            <tunnel_inside_address>
-              <ip_address>169.254.44.41</ip_address>
-              <network_mask>255.255.255.252</network_mask>
-              <network_cidr>30</network_cidr>
-            </tunnel_inside_address>
-            <bgp>
-              <asn>7224</asn>
-              <hold_time>30</hold_time>
-            </bgp>
-            </vpn_gateway>
-            <ike>
-            <authentication_protocol>sha1</authentication_protocol>
-            <encryption_protocol>aes-128-cbc</encryption_protocol>
-            <lifetime>28800</lifetime>
-            <perfect_forward_secrecy>group2</perfect_forward_secrecy>
-            <mode>main</mode>
-            <pre_shared_key>Iw2IAN9XUsQeYUrkMGP3kP59ugFDkfHg</pre_shared_key>
-            </ike>
-            <ipsec>
-            <protocol>esp</protocol>
-            <authentication_protocol>hmac-sha1-96</authentication_protocol>
-            <encryption_protocol>aes-128-cbc</encryption_protocol>
-            <lifetime>3600</lifetime>
-            <perfect_forward_secrecy>group2</perfect_forward_secrecy>
-            <mode>tunnel</mode>
-            <clear_df_bit>true</clear_df_bit>
-            <fragmentation_before_encryption>true</fragmentation_before_encryption>
-            <tcp_mss_adjustment>1387</tcp_mss_adjustment>
-            <dead_peer_detection>
-              <interval>10</interval>
-              <retries>3</retries>
-            </dead_peer_detection>
-            </ipsec>
-          </ipsec_tunnel>
-        </vpn_connection>
+    """ + escape(CUSTOMER_GATEWAY_CONFIGURATION_TEMPLATE) + \
+    """
       </customerGatewayConfiguration>
       <type>ipsec.1</type>
       <customerGatewayId>{{ vpn_connection.customer_gateway_id }}</customerGatewayId>
-      <vpnGatewayId>{{ vpn_connection.vpn_gateway_id }}</vpnGatewayId>
+      <vpnGatewayId> {{ vpn_connection.vpn_gateway_id or '' }} </vpnGatewayId>
```

Comment:
nitpick: we could remove the whitespaces around the gateway ID value:
```
<vpnGatewayId>{{ vpn_connection.vpn_gateway_id or '' }}</vpnGatewayId>
```
(not critical, though)